src/Voron/Impl/FreeSpace/FreeSpaceHandling.cs

```diff
@@ -192,19 +194,31 @@ public void Clear()
             {
                 var current = new StreamBitArray(it.CreateReaderForCurrent());
```

Comment:
Better to move this _after_ the dictionary check, saves the work if we skip it.

Reply:
Done